### ISSUE TITLE
Training loss of  llm becomes NaN

### ISSUE THREAD

**Issue by zwei-lin** [iSEE-Laboratory/LLMDet#18]
Hi,

Thanks for the great work. 
I have some problems when using the tiny/base/large.pth. I try to load from tiny/base/large.pth with default configs (grounding_dino_swin_t/b/l) and want to continue to train for more iterations. However, the loss of llm becomes NaN at the first iteration. Is there any trick to load llm weights?

Thanks again.
